app/scripts/components/scenariobuilder.js

```diff
@@ -233,12 +233,19 @@ var exports = (function() {
       updateAfterTick: function() {
 
         if ($("[data-toggle='tooltip']").tooltip !== undefined) {
-
           $("[data-toggle='tooltip']").tooltip({
             html: true,
-            // hover activation annoys some people
+            // do not user hover: we use clickable links in the tooltip
             trigger: "click"
           });
+
+          // register event for closing tooltips when clicking anywhere in the body
+          $("body").click(function (evt) {
```

Comment:
- html can be bigger than body $("html").click
- you can also consider using $("html").one. This would be important if mulltiple tooltips are open,, so that the event is only called once per tooltip